lib/xenopsd.ml

```diff
@@ -190,13 +190,15 @@ let rpc_fn call =
             name= "VM.import_metadata"
           ; params=
               [Rpc.Dict [("debug_info", debug_info); ("metadata", metadata)]]
+          ; notif= false
```

Comment:
Referring to my comment [here](https://github.com/xapi-project/xcp-idl/pull/314#pullrequestreview-499372010), maybe we should bite the bullet and upgrade to rpclib v8 now?